tests/fixtures/issues/issue-1219.php

```diff
@@ -6308,59 +6308,6 @@
         'lite' => false,
         'standard' => false,
     ],
-    'issue-1219-119' => [
-        'ua' => 'Mozilla/5.0 (Windows; U; Windows NT 6.1; en-GB; rv:1.9.1.17) Gecko/20110123 Mozilla/5.0 (Windows; U; Windows NT 5.1; en-US; rv:1.8.1.2) Gecko/20070225 lolifox/0.32',
```

Comment:
Ah, no, it came from here apparently:

https://github.com/zsxsoft/php-useragent/blob/f8bf32db3064becdc6d3fac9be61260cd7e1f1bd/tests/UserAgentList.php#L1303

I could leave it in and make it match zsxsoft's test case. Seems that other parsers disagree on how it should be parsed:

http://useragent.mkf.solutions/?userAgent=Mozilla%2F5.0+%28Windows%3B+U%3B+Windows+NT+6.1%3B+en-GB%3B+rv%3A1.9.1.17%29+Gecko%2F20110123+Mozilla%2F5.0+%28Windows%3B+U%3B+Windows+NT+5.1%3B+en-US%3B+rv%3A1.8.1.2%29+Gecko%2F20070225+lolifox%2F0.32